resources/rest-service/cloudify/migrations/versions/03ad040e6f78_6_1_to_6.2.py

```diff
@@ -139,67 +128,89 @@ def _drop_audit_log_table():
+                    VALUES (_table, NEW._storage_id, 'create', _user, _execution_id, now());
+                RETURN NEW;
+            ELSEIF (TG_OP = 'UPDATE') THEN
+                INSERT INTO audit_log (ref_table, ref_id, operation, creator_name, execution_id, created_at)
+                    VALUES (_table, NEW._storage_id, 'update', _user, _execution_id, now());
+                RETURN NEW;
+            ELSEIF (TG_OP = 'DELETE') THEN
+                INSERT INTO audit_log (ref_table, ref_id, operation, creator_name, execution_id, created_at)
+                    VALUES (_table, OLD._storage_id, 'delete', _user, _execution_id, now());
+                RETURN OLD;
+            END IF;
+            RETURN NULL;
+        END;
+    $$ LANGUAGE plpgsql;
+
+
+    CREATE OR REPLACE FUNCTION write_audit_log_id() RETURNS TRIGGER AS $$
+        DECLARE
+            _table TEXT := TG_ARGV[0]::TEXT;
+            _user TEXT := audit_username();
+            _execution_id TEXT := audit_execution_id();
+        BEGIN
+            IF (TG_OP = 'INSERT') THEN
+                INSERT INTO audit_log (ref_table, ref_id, operation, creator_name, execution_id, created_at)
+                    VALUES (_table, NEW.id, 'create', _user, _execution_id, now());
+                RETURN NEW;
+            ELSEIF (TG_OP = 'UPDATE') THEN
+                INSERT INTO audit_log (ref_table, ref_id, operation, creator_name, execution_id, created_at)
+                    VALUES (_table, NEW.id, 'update', _user, _execution_id, now());
+                RETURN NEW;
+            ELSEIF (TG_OP = 'DELETE') THEN
                 INSERT INTO audit_log (ref_table, ref_id, operation, creator_name, execution_id, created_at)
-                    VALUES (_table, _id, 'delete', _user, _execution, now());
+                    VALUES (_table, OLD.id, 'delete', _user, _execution_id, now());
                 RETURN OLD;
-            ELSE
-                IF (_table = 'blueprints_labels') THEN
-                    _id = NEW.id;
-                ELSE
-                    _id = NEW._storage_id;
-                END IF;
-                IF (TG_OP = 'UPDATE') THEN
-                    INSERT INTO audit_log (ref_table, ref_id, operation, creator_name, execution_id, created_at)
-                        VALUES (_table, _id, 'update', _user, _execution, now());
-                    RETURN NEW;
-                ELSIF (TG_OP = 'INSERT') THEN
-                    INSERT INTO audit_log (ref_table, ref_id, operation, creator_name, execution_id, created_at)
-                        VALUES (_table, _id, 'create', _user, _execution, now());
-                    RETURN NEW;
-                END IF;
             END IF;
             RETURN NULL;
         END;
-        $func$ LANGUAGE plpgsql;""")
+    $$ LANGUAGE plpgsql;""")
 
 
-def _drop_function_write_audit_log():
+def _drop_functions_write_audit_log():
     op.execute("""DROP FUNCTION write_audit_log();""")
```

Comment:
Noting that this is missing the other DROP FUNCTION calls it should have, just in case you didn't include them in a later commit.